packages/manager/modules/dbaas-logs/src/logs/empty/empty.html

```diff
@@ -0,0 +1,52 @@
+<div class="container mx-0 px-0 ldp-project-empty">
+    <div class="row align-items-center">
+        <div class="col-md-4">
+            <div data-ng-class="{ 'placeholder p-5' : !$ctrl.imageSource }">
+                <img
+                    data-ng-src="{{:: $ctrl.imageSource || $ctrl.placeholder }}"
+                    alt="{{::'alt_illustration' | translate}}"
```

Comment:
Since translation is empty then you can leave the attribute empty.

```suggestion
                    alt=""
```